### PR TITLE
Bugfix

### PR DESCRIPTION
**Pull request by husky-parul**
This PR includes:
    - [Bug fix](https://github.com/sustainable-computing-io/kepler-operator/issues/51) to export port from controller to crd
    - Updates README.MD and Contribution Guidelines
    - Provides manifests to deploy Kepler's default Grafana dashboard
    - Provides script for local kind cluster

### CODE REVIEW

File: api/v1alpha1/kepler_types.go [hunk 1]
```diff
@@ -41,6 +41,8 @@ type Sources struct {
 
 type CollectorSpec struct {
 	Image string `json:"image,omitempty"`
+	// +kubebuilder:default=9102
```
Comment (rootfs) [inline api/v1alpha1/kepler_types.go:44]: the default and sample ports are different. Let's use 9103 for consistency.

**Comment by rootfs** [comment]
@husky-parul does #53 have any conflict with this PR?

**Comment by husky-parul** [comment]
> @husky-parul does #53 have any conflict with this PR?

I see [53](https://github.com/sustainable-computing-io/kepler-operator/pull/53) only removes the use of `hostNetwork` and `hostPort`. So there shoul be no conflict. I will review and merge the PR. once https://github.com/sustainable-computing-io/kepler-operator/pull/52 is merged.

File: config/crd/bases/kepler.system.sustainable.computing.io_keplers.yaml [hunk 2]
```diff
@@ -37,6 +37,9 @@ spec:
             properties:
               collector:
                 properties:
+                  collectorPort:
+                    default: 9102
```
Comment (rootfs) [inline config/crd/bases/kepler.system.sustainable.computing.io_keplers.yaml:41]: would this be the default?

Comment (husky-parul) [reply config/crd/bases/kepler.system.sustainable.computing.io_keplers.yaml:41]: That's from the previous commit. `make generate` and `make manifests` should have updated this.
I will fix it

**Review by rootfs — APPROVED** [review]

**Pull request merged**